src/rhsm/profile.py

```diff
@@ -86,7 +86,8 @@ def __generate():
                             "version": module_pkg.getVersion(),
                             "context": module_pkg.getContext(),
                             "arch": module_pkg.getArch(),
-                            "profiles": [profile.getName() for profile in module_pkg.getProfiles()]
+                            "profiles": [profile.getName() for profile in module_pkg.getProfiles()],
+                            "installed_profiles": modules.getInstalledProfiles(module_pkg.getName())
```

Comment:
How should I understand to this PR? Is `profiles` useless for you and you need content provided in `installed_profiles`. I'm not against adding another item to this list.

Reply:
while ```profiles``` itself is not as useful as ```installed_profiles``` I am ok with having that information as an extra.

Reply:
@jirihnidek @parthaa This looks good to me. Anything left before we merge?

Reply:
nope I am happy with this :)